vector/src/main/java/im/vector/app/features/home/room/detail/composer/VoiceMessageHelper.kt

```diff
@@ -169,11 +171,19 @@ class VoiceMessageHelper @Inject constructor(
     }
 
     fun stopPlayback() {
-        playbackTracker.stopPlayback(VoiceMessagePlaybackTracker.RECORDING_ID)
+        playbackTracker.pausePlayback(VoiceMessagePlaybackTracker.RECORDING_ID)
         mediaPlayer?.stop()
         stopPlaybackTicker()
     }
 
+    fun movePlaybackTo(id: String, percentage: Float, totalDuration: Int) {
```

Comment:
if we use the `mediaPlayer?.duration` then we'll be able to avoid inferring the duration from the UI countdown ticker 
would that be possible? (it would assume the media player is in the prepared state)

Reply:
We have a single mediaplayer which is initialized when users start the playback. But users are able to scroll idle recordings and the mediaplayer is null at this stage.